.prettierrc

```
@@ -0,0 +1,16 @@
+{
+  "tabWidth": 2,
+  "semi": false,
+  "singleQuote": true,
+  "useTabs": false,
+  "bracketSpacing": true,
+  "bracketSameLine": false,
+  "arrowParens": "avoid",
+  "endOfLine": "auto",
+  "trailingComma": "all",
+  "printWidth": 100,
+  "quoteProps": "as-needed",
+  "embeddedLanguageFormatting": "auto",
+  "htmlWhitespaceSensitivity": "css",
+  "jsxSingleQuote": true,
+}
```

Half of the options have default values

```suggestion
{
  "semi": false,
  "singleQuote": true,
  "arrowParens": "avoid",
  "endOfLine": "auto",
  "printWidth": 100,
  "jsxSingleQuote": true,
}
```